src/main/webapp/view.js

```diff
@@ -32,3 +35,11 @@ function getLectureId() {
   const urlParams = new URLSearchParams(window.location.search);
   return urlParams.get(PARAM_ID);
 }
+
+/**
+ * Returns the video id obtained from {@code window.location}.
```

Comment:
If someone who do not know how this class is implemented, would they understand what window.location is?

Reply:
made this clearer